packages/types/src/types.ts

```diff
@@ -110,7 +110,7 @@ export interface GuidedAnswerAPI {
 export interface GuidedAnswersFeedback {
     treeId: GuidedAnswerTreeId;
     nodeId: GuidedAnswerNodeId;
-    message: 'Solved' | 'Not Solved' | string;
+    message: string; // if used to sent feedback outcome message is set to  'Solved' | 'Not Solved'
```

Comment:
Please change the comment. Doesn't sound right.

Reply:
@theStiggySpd, you are right it sounds confusing. I removed the comment completely. The fact that it needs to be set to `Solved` or `Nor Solved` is implemented  behind the API function and the API expects `solved: boolean`, so it is clear to consumers and does not need additional comments:
https://github.com/SAP/guided-answers-extension/blob/6419ba932f868a66ad4d5c7110db85e696bef81a/packages/core/src/guided-answers-api.ts#L302-L310

Reply:
yes i saw that. Thanks